package.json

```diff
@@ -54,12 +54,14 @@
     "zod": "^3.x"
   },
   "devDependencies": {
+    "@jest/types": "^29.5.0",
```

Comment:
@ecyrbe For thoses version you want exact version or you accept later versions ?

Reply:
Oh, yes exact versions. To have precise control on the updates. I use renovate bot to handle updates for me. It allows to detect breaking changes in dependencies with a PR.